modules/setting/mime_type_map.go

```
@@ -21,6 +24,7 @@ func newMimeTypeMap() {
 	m := make(map[string]string, len(keys))
 	for _, key := range keys {
 		m[strings.ToLower(key.Name())] = key.Value()
+		_ = mime.AddExtensionType(key.Name(), key.Value())
```

I don't think we should ignore a error here.